.changeset/sharp-dragons-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@techdocs/cli': patch
+---
+
+Use a local file dependency for embedded-techdocs-app, to ensure that it's always pulled out of the workspace
```

Comment:
No need for a changeset really though, this only affects the main repo

Reply:
I removed it

Reply:
thanks!